sunpy/io/tests/test_fits.py

```diff
@@ -124,3 +125,14 @@ def test_write_with_metadict_header_astropy(tmpdir):
     temp_file = tmpdir / "temp.fits"
     sunpy.io.fits.write(str(temp_file), data, meta_header)
     assert temp_file.exists()
+
+
+def test_fitsheader():
+    extensions = ('fts', 'fits')
```

Comment:
Add a short comment here explaining what this test is doing? (Testing all the sample data parses???)